src/status_im/ui2/screens/communities/communities_home.cljs

```diff
@@ -101,13 +103,26 @@
                                 :description         (i18n/label :t/whats-trending)
                                 :accessibility-label :communities-home-discover-card}])
 
+
+(defn community-card []
+    [community-card/community-card {
+                                     :title  (i18n/label :t/decent-title)
+                                     :description (i18n/label :t/decent-desc)
+                                     :image (resources/get-image :decentraland)
+                                     :member_count (i18n/label :t/member-count)
+                                     :likes_count (i18n/label :t/likes-count)
+                                     :dislike_count (i18n/label :t/dislike-count)
+                                     }])
+
+
 (defn communities-home []
   (let [communities  (<sub [:communities/communities])]
     [rn/view  {:flex    1}
      [topnav/top-nav {:type    :default}]
      [title-column]
      [discover-card]
      [home-community-segments]
+     [community-card]
```

Comment:
Add your component for preview to `status-im.ui2.screens.quo2-preview.` It should not be here.
See examples of how other components in for instance: 
`status-im.ui2.screens.quo2-preview.community.community-card-view` are added to quo2_preview.